tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -981,15 +990,20 @@ const reducer = createReducer(
   on(actions.rangeSelectionToggled, (state) => {
     const nextRangeSelectionEnabled = !state.rangeSelectionEnabled;
     let nextStepSelectorEnabled = state.stepSelectorEnabled;
+    let linkedTimeSelection = state.linkedTimeSelection;
 
     if (nextRangeSelectionEnabled) {
       nextStepSelectorEnabled = nextRangeSelectionEnabled;
+      if (!linkedTimeSelection || !linkedTimeSelection.end) {
+        linkedTimeSelection = getMinMaxTimeSelection(state.timeSeriesData);
```

Comment:
I mean, can I just reuse the value stepMinMax here with perhaps some infinity check? If not, why?
```
linkedTimeSelection = {
start: stepMinMax.min,
end: stepMinMax.max,
}
```

Reply:
I didn't know that property was already computed. That would have saved me hours.